drivers/display/display_st7735r.c

```diff
@@ -67,32 +67,37 @@ static void st7735r_set_lcd_margins(const struct device *dev, uint16_t x_offset,
 	data->y_offset = y_offset;
 }
 
-static void st7735r_set_cmd(const struct device *dev, int is_cmd)
+static int send_packet(const struct device *dev, uint32_t is_data, const uint8_t *tx_data,
+		       size_t tx_count)
 {
+	int ret = 0;
 	const struct st7735r_config *config = dev->config;
+	struct spi_buf tx_buf;
+	struct spi_buf_set tx_bufs = {.buffers = &tx_buf, .count = 1};
+
+		tx_buf.buf = (void *)tx_data;
+		tx_buf.len = tx_count;
+
+		/* Drive Data/Comman pin  */
+		gpio_pin_set_dt(&config->cmd_data, (is_data ? 0 : 1));
```

Comment:
> Some additional tab added in some lines to make next commit more clear.

Don't do this. Each commit should be able to pass code style checks

Reply:
Thanks Maureen for feedback.
Sure.